Mathlib/Analysis/Normed/Lp/ProdLp.lean

```diff
@@ -852,6 +852,17 @@ def prodEquivₗᵢ : WithLp ∞ (α × β) ≃ₗᵢ[𝕜] α × β where
 
 end IsBoundedSMul
 
+instance instProdNormSMulClass [SeminormedRing 𝕜] [Module 𝕜 α] [Module 𝕜 β]
```

Comment:
Can we get this cheaply from the `Pi` version using `prodEquivPiFinTwo`?

Reply:
... aargh, no we can't, because `prodEquivPiFinTwo` requires `α` and `β` to live in the same type universe while the product constructor doesn't. So please ignore this suggestion.

Reply:
Resolving this to ignore it as instructed :)